packages/core/src/aws-sdk-helpers/cloudFront/fetchDistributionConfig.ts

```diff
@@ -4,7 +4,7 @@ import {
 } from '@aws-sdk/client-cloudfront';
 import { cloudFrontClient } from '../../clients';
 
-const getDistributionConfig = async (
+const fetchDistributionConfig = async (
```

Comment:
I think this change is breaking the rule "cloudFrontNoHttp" due to recent conflicts

Reply:
Fixed